Cargo.toml

```diff
@@ -98,7 +98,7 @@ signal-hook-tokio = "0.3.1"
 sqlx = "0.7.4"
 stopper = "0.2.7"
 tempfile = "3.10.1"
-testcontainers = "0.15.0"
+testcontainers = { version = "0.16.3", features = ["blocking"] }
```

Comment:
Is there a good reason not to use the `AsyncRunner` everywhere -- it looks like you already used it everywhere except for `EphemeralDatabase`? Dropping the use of `SyncRunner` would let us drop the `blocking` feature.

Reply:
No, I was thinking of doing that as a follow-up once everything was working